maintainers/maintainer-list.nix

```diff
@@ -13718,6 +13718,12 @@
     githubId = 66133083;
     name = "Tomas Bravo";
   };
+  tchab = {
```

Comment:
The maintainers list deserves its own commit.

No need for a PR, just split this in a new commit with the title `maintainers: add tchab`.

Tutorial: https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits

Reply:
Should be ok now.